apps/masterbots.ai/lib/hooks/use-mb-chat.ts

```
@@ -241,44 +252,53 @@ export function useMBChat(config?: MBChatHookConfig): MBChatHookCallback {
+          chatbotId: chatbot.chatbotId,
+          jwt: session!.user?.hasuraJwt,
+          userId: session!.user.id,
+          isPublic: activeChatbot?.name !== 'BlankBot'
+        })
 
-      updateActiveThread(thread)
-    }
+        // * Loading: Here is the information you need... 'finish'
+        const thread = await getThread({
+          threadId: threadId as string,
+          jwt: session!.user?.hasuraJwt
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)